### PR TITLE
SO-112: Install helmcharts via CSO

### PR DESCRIPTION
**Pull request by fbm3307**
This PR is to install Helmcharts via CSO as part of Downsizing phase-2

### CODE REVIEW

File: pkg/stub/handler.go [hunk 1]
```diff
@@ -6,25 +6,19 @@ import (
...
-	kapis "k8s.io/apimachinery/pkg/apis/meta/v1"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	utilnet "k8s.io/apimachinery/pkg/util/net"
-	"k8s.io/apimachinery/pkg/util/wait"
-	corev1client "k8s.io/client-go/kubernetes/typed/core/v1"
-	corev1lister "k8s.io/client-go/listers/core/v1"
-	restclient "k8s.io/client-go/rest"
-	"k8s.io/client-go/util/retry"
+	"golang.org/x/mod/semver"
+	"gopkg.in/yaml.v2"
```
Comment (filariow) [inline pkg/stub/handler.go:21]: The following links may be useful

* https://github.com/kubernetes-sigs/yaml
* https://pkg.go.dev/sigs.k8s.io/yaml

File: pkg/stub/handler.go [hunk 2]
```diff
@@ -1393,3 +1453,161 @@ func (h *Handler) numOfManagedImageStreamsPresent() int {
...
+	indexURL := "https://charts.openshift.io/index.yaml"
+
+	resp, err := http.Get(indexURL)
+	if err != nil {
+		return nil, err
+	}
+	if resp.StatusCode != 200 {
+		return nil, errors.New(fmt.Sprintf("Response for %v returned %v with status code %v", indexURL, resp, resp.StatusCode))
+	}
+	body, err := ioutil.ReadAll(resp.Body)
```
Comment (filariow) [inline pkg/stub/handler.go:1488]: https://pkg.go.dev/io/ioutil#ReadAll

File: pkg/stub/handler.go [hunk 3]
```diff
@@ -1393,3 +1453,161 @@ func (h *Handler) numOfManagedImageStreamsPresent() int {
...
+				if k1 == "name" {
+					chartsValue.Name = fmt.Sprintf("%v", v1)
+				}
+				if k1 == "version" {
+					chartsValue.Version = fmt.Sprintf("%v", v1)
+				}
+				if k1 == "urls" {
+					chartsValue.churl = fmt.Sprintf("%v", v1)
+					chartsValue.churl = chartsValue.churl[1 : len(chartsValue.churl)-1]
+				}
```
Comment (filariow) [inline pkg/stub/handler.go:1517]: maybe using the switch statement here may be a good idea

File: pkg/stub/handler.go (hunk 2 above)
Comment (fbm3307) [reply pkg/stub/handler.go:1489]: done

File: pkg/stub/handler.go (hunk 1 above)
Comment (fbm3307) [reply pkg/stub/handler.go:22]: done

**Pull request closed without merging**